utility_network/display-utility-associations/build.gradle

```
@@ -0,0 +1,77 @@
+plugins {
+    id 'application'
+    id 'org.openjfx.javafxplugin' version '0.0.5'
+}
+
+group = 'com.esri.samples'
+
+ext {
+    arcgisVersion = '100.8.0'
+}
+
+javafx {
+    version = "11.0.1"
+    modules = [ 'javafx.controls', 'javafx.fxml' ]
```

since we're not making use of a .fxml file for this sample, the fxml module can be removed here
```suggestion
    modules = [ 'javafx.controls' ]
```